includes/admin/class-sensei-onboarding.php

```diff
@@ -32,7 +32,10 @@ class Sensei_Onboarding {
 	 * @var array
 	 */
 	private $user_data_defaults = [
-		'features'  => [],
+		'features'  => [
+			'selected' => [],
+			'options'  => [],
```

Comment:
The `options` is not saved, should not be in here.
Also, bumping the `__version` (`1-dev2`) when the data structure changes will make sure that older data is cleared, avoiding errors when we test.

Reply:
@donnapep as a workaround for the error, you can clear the `sensei_setup_wizard_data` option from the database.

Reply:
Oops! My bad! Thank you for the catch @yscik! Probably it was the problem @donnapep. Could you test if it fixed?

Fixed here: https://github.com/Automattic/sensei/pull/3190/commits/3062621856d3dfe1528f1061ebbed928070cddbf

Obs.: I already updated the version to `1`, because probably we'll not change more this structure and we could forget to update it before to release. Make sense to you? Another approach could be to create a card with reminders to the release.